src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZUtilsTest.java

```diff
@@ -0,0 +1,487 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.buildSendString;
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.oneByteToTwoBytes;
+import static org.junit.Assert.assertArrayEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.Category;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.PayloadType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.Command;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.CommandType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.AEMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BacklightStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PanTiltDrive;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PowerStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PresetControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.RGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowPanTiltStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowShutterStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.ZoomControl;
+
+/**
+ * Unit test for AverPTZ Utils - method build send string
+ * Build send string which match expected string
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZUtilsTest {
+	int cameraID = 1;
+	int sequenceNumber = 1;
+
+	/**
+	 * Test AverPTZUtils#buildSendString success
+	 * Expect build send string for control command which match expected string
+	 */
+	@Test
+	public void testBuildControlCommandSendString() throws IOException {
```

Comment:
the UT should be assert as a smallest unit. 
With this UT, we can fail at the first case and the rest won't be executed. So suggest to separate them.